### PR TITLE
team update

### LINKED ISSUE #404: Update with New Community Members
Update the `src/contents/members` folder with new community members mardown file

### PR DESCRIPTION
**Pull request by srajankumar**
Updates the new team with the community leads

Closes #404

### CODE REVIEW

File: src/contents/members/Akshess.md
Comment (dev-shetty) [inline src/contents/members/Akshess.md:1]: Akshay is current senior

File: src/contents/members/Ananya.md
Comment (dev-shetty) [inline src/contents/members/Ananya.md:1]: Ananya is also current senior

File: src/contents/members/anish.md
Comment (dev-shetty) [inline src/contents/members/anish.md:1]: Anish is current senior

File: src/contents/members/gauravraoj.md
Comment (dev-shetty) [inline src/contents/members/gauravraoj.md:1]: Gaurav is also current senior not Alumni

File: src/contents/members/pavan.md
Comment (dev-shetty) [inline src/contents/members/pavan.md:1]: Pavan is current senior

File: src/contents/members/vinayak.md
Comment (dev-shetty) [inline src/contents/members/vinayak.md:1]: Vinayak is current Senior as well

**Review by dev-shetty — CHANGES_REQUESTED** [review]
I have put some reviews regarding the roles just go through that once

**Comment by srajankumar** [comment]
> I have put some reviews regarding the roles just go through that once

so, should i put them in Members?

**Comment by dev-shetty** [comment]
> > I have put some reviews regarding the roles just go through that once
> 
> so, should i put them in Members?

Asking seniors.....will let u know

**Review by dev-shetty — APPROVED** [review]
LGTM ✅

**Review by dev-shetty — APPROVED** [review]

**Pull request merged**